wpeverest/everest-forms
# Feature - Google calendar integration for appointment scheduling

### All Submissions:


### Changes proposed in this Pull Request:

This PR introduces the new feature **Google Calendar Integration**. We can create a new event in the Google Calendar which is useful for **appointment scheduling**.
Google API and Services link:- https://console.cloud.google.com/apis
Follow the docs link for Google Drive integration if you are unsure how to create the API. The process will be the same for Google Calendar, but you need to enable Google Calendar from the library tab, which will be different.
**Link:-** https://docs.everestforms.net/docs/how-to-upload-files-to-google-drive-or-dropbox/

### Dependancy PR:
https://github.com/wpeverest/everest-forms-pro/pull/840

### How to test the changes in this Pull Request:

1. Create keys to connect the account from the Google API and Services console.
2. Goto Global settings > Integration tab > google calendar > put your keys and authenticate.
3. After that create a form insert date field > field option > Appointment scheduling > Enable Google Calendar
4. Choose the Event title and description field
5. Choose a date, submit the form, and verify the new event on Google Calendar
6. Also check for all the possible cases of date fields like date, time, both, different modes

### Types of changes:


### Other information:


### Changelog entry

> Feature - Google calendar integration for appointment scheduling.
